Makefile

```diff
@@ -177,7 +177,7 @@ lint-sol: ## run linter for Solidity
 .PHONY: lint-rust
 lint-rust: ## run linter for Rust
 	cargo fmt --check
-	#cargo clippy -- -Dwarnings			# cannot be used unless all deprecation notices are removed as well
+	cargo clippy -- -Dwarnings
```

Comment:
Why not `cargo clippy -- -Dwarnings -Adeprecated` and keep the RLP deprecation in, so it pops up when we build locally?